server/odc-core/src/main/java/com/oceanbase/odc/core/alarm/AlarmEvent.java

```diff
@@ -16,23 +16,46 @@
 
 package com.oceanbase.odc.core.alarm;
 
+import java.util.Map;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
 
 @Getter
 public class AlarmEvent {
 
     private final String eventName;
 
-    private final String eventMessage;
+    private final AlarmMessage eventMessage;
 
     private final AlarmLevel level;
 
-    public AlarmEvent(String eventName, String eventMessage, AlarmLevel level) {
+    public AlarmEvent(String eventName, AlarmMessage eventMessage, AlarmLevel level) {
         this.eventName = eventName;
         this.eventMessage = eventMessage;
         this.level = level;
     }
 
+    @AllArgsConstructor
+    @NoArgsConstructor
+    @Data
+    @Accessors(chain = true)
+    public static class AlarmMessage {
```

Comment:
it is concept in ocp, should not involved in odc

Reply:
changed